File: python/PetShop.py
#deklarasi class
class PetShop():
    #atribut private
    __id = ""
    __nama = ""
    __harga = ""
    __stok = ""

    #constuctor
    def __init__(self, id = "", nama = "", harga = "", stok = ""):
        self.id = id
        self.nama = nama
        self.harga = harga
        self.stok = stok

    #setter
    def set_id(self, id):
        self.__id = id
    def set_nama(self, nama):
        self.__nama = nama
    def set_harga(self, harga):
        self.__harga = harga
    def set_stok(self, stok):
        self.__stok = stok

    #getter
    def getid(self):
        return self.__id
    def getnama(self):
        return self.__nama
    def getharga(self):
        return self.__harga
    def getstok(self):
        return self.__stok

File: php/PetShop.php
<?php
//deklarasi class
class PetShop
{
    //private atribut
    private $id;
    private $nama;
    private $harga;
    private $stok;
    private $foto;

    //constructor
    public function __construct($id = '', $nama = '', $harga = '', $stok = '',$foto = '')
    {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->foto = $foto;
    }

    //setter
    public function setid($id)
    {
        $this->id = $id;
    }
    public function setnama($nama)
    {
        $this->nama = $nama;
    }
    public function setharga($harga)
    {
        $this->harga = $harga;
    }
    public function setstok($stok)
    {
        $this->stok = $stok;
    }
    public function setfoto($foto)
    {
        $this->foto = $foto;
    }

    //getter
    public function getid()
    {
        return $this->id;
    }
    public function getnama()
    {
        return $this->nama;
    }
    public function getharga()
    {
        return $this->harga;
    }
    public function getstok()
    {
        return $this->stok;
    }
    public function getfoto()
    {
        return $this->foto;
    }
}
?>